ChangeLog

```diff
@@ -1,4 +1,4 @@
-2021-01-06  Conrad Sanderson  <conrad.sanderson.nospam@nogmail.com>
+2021-01-06  Conrad Sanderson  <cs.rcpparmadillo@gmail.com>
```

Comment:
The point is to have an address that identifies you in case someone wants to get hold of you.  So, no, a freshly created tar pit will not do.  

You are not required to file PRs here.  If you don;t want to fine.  But please respect that I have adhered to very common (if GNU-ish) standards here for well over a decade, and I am not going to stop now.

Reply:
It's not a "tar pit".  That's a perfectly fine email address which is valid and works. Emails sent to that address will get to me.  

The point of this email address is to have a separate and unique channel for RcppArmadillo related stuff.

Reply:
Well, I am repo maintainer and I take the other view here.